rcl/test/rcl/test_subscription.cpp

```diff
@@ -152,12 +150,34 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
   subscription = rcl_get_zero_initialized_subscription();
   EXPECT_FALSE(rcl_subscription_is_valid(&subscription));
   rcl_reset_error();
+}
 
-  // Check that valid subscriber is valid
-  subscription = rcl_get_zero_initialized_subscription();
+/* Basic nominal test of a subscription: valid subscriber is valid.
+ */
+TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription_nominal_valid) {
```

Comment:
```suggestion
TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription_nominal) {
```

Also, please change the name of the previous test so it more closely matches what it is testing. I suggest:

`test_subscription_init_fini_and_is_valid`

Reply:
Thanks, done by 3173e8790dc361a4b1ccace8e7710ebf7a863d64.
To change both , I didn't use "Commit suggestion".